### PR TITLE
Belong to easy-archive-bag issue 38

### PR DESCRIPTION
**Pull request by ekoi**
I am not sure about my solution.

### CODE REVIEW

**Comment by janvanmansum** [comment]
What are you uncertain about in particular?

**Comment by ekoi** [comment]
Actually, many things. One of my concerns, I do the following:
if state == STATE_SUBMITTED
--
--
I would like to log when the state is not SUBMITTED but I don't know how.

File: src/main/assembly/dist/cfg/application.properties [hunk 1]
```diff
@@ -7,6 +7,7 @@ fcrepo.user={{ easy_ingest_flow_fcrepo_user }}
 fcrepo.password={{ easy_ingest_flow_fcrepo_password }}
 sync.num-tries={{ easy_ingest_flow_sync_num_tries }}
 sync.delay={{ easy_ingest_flow_sync_delay }}
+check.interval={{ easy_ingest_flow_check_inteval }}
```
Comment (janvanmansum) [inline src/main/assembly/dist/cfg/application.properties]: typo: "interval" missing an "r"

**Comment by janvanmansum** [comment]
@ekoi You could add a log line before the state is checked.

File: src/main/scala/nl/knaw/dans/easy/ingest_flow/EasyIngestFlow.scala [hunk 2]
```diff
@@ -104,7 +107,8 @@ object EasyIngestFlow {
       _ <- assertNoAccessSet(xml)
       urn <- requestUrn()
       (doi, otherAccessDOI) <- getDoi(xml)
-      storageDatasetDir <- archiveBag()
+      (storageDatasetDir, state) <- archiveBag()
```
Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/ingest_flow/EasyIngestFlow.scala]: Maybe add a log statement about the returned state here:

    _ = log.info(s"Archival storage service returned: $state")

Comment (ekoi) [inline src/main/scala/nl/knaw/dans/easy/ingest_flow/EasyIngestFlow.scala]: Ok, we can do that but I would like to add log.warn (or log.error ?) when the state is other than submitted.

**Pull request merged**